package.json

```diff
@@ -20,6 +21,34 @@
     "storybook": "start-storybook -s ./",
     "deploy-storybook": "storybook-to-ghpages && ./.storybook/pushFonts.sh"
   },
+  "jest": {
+    "collectCoverage": true,
+    "transform": {
+      "^.+\\.tsx?$": "ts-jest"
+    },
+    "coverageThreshold": {
+      "global": {
+        "branches": 15,
+        "functions": 15,
+        "lines": 15,
+        "statements": 15
+      }
+    },
+    "coverageReporters": [
+      "json",
+      "lcov",
+      "text",
+      "clover",
```

Comment:
What's clover?

Reply:
https://istanbul.js.org/docs/advanced/alternative-reporters/

Reply:
Removed some of them and kept `text` and `html` which you can see by running `open coverage/index.html`